packages/renderer/src/screenshot-task.ts

```diff
@@ -75,7 +75,12 @@ export const screenshotTask = async ({
 								width,
 							},
 				captureBeyondViewport: true,
-				optimizeForSpeed: true,
+				optimizeForSpeed: false,
```

Comment:
@JonnyBurger This change made my renders drastically slower, was it intentional?